src/main/java/de/zalando/aruha/nakadi/repository/EventTypeRepository.java

```diff
@@ -18,4 +19,14 @@
     List<EventType> list();
 
     void removeEventType(String name) throws InternalNakadiException, NoSuchEventTypeException;
+
+    default Optional<EventType> findByNameO(final String eventTypeName) throws InternalNakadiException {
```

Comment:
This alternative has been considered previously but we decided to use throw/catch style. I see no advantage in introducing this now.